QualityControl/Dockerfile

```diff
@@ -16,17 +21,38 @@ RUN apk add --no-cache \
     giflib-dev \
     musl-dev
 
-# Working directory
-WORKDIR /usr/src/app
+# Export port
+EXPOSE 8080
 
-# Install dependencies
-COPY . .
+FROM base AS dependencies
+
+# Install dependencies early for Docker layer caching
+COPY package*.json ./
 RUN npm ci --silent
-RUN npm i -g nodemon --save-dev
 
+# Copy the rest of the application code
+COPY . .
 
-# Export port
-EXPOSE 8080
+FROM dependencies AS chromiumdependencies
+# Required for Puppeteer
+RUN apk add --no-cache \
```

Comment:
I think for dev, it might be good to use the cache options, otherwise it takes quite a while to setup